modulus/models/sfno/sfnonet.py

```diff
@@ -737,7 +736,7 @@ def _init_weights(self, m):
             trunc_normal_(m.weight, std=0.02)
             if m.bias is not None:
                 nn.init.constant_(m.bias, 0)
-        elif isinstance(m, nn.LayerNorm) or isinstance(m, FusedLayerNorm):
+        elif isinstance(m, nn.LayerNorm):
```

Comment:
I might be overthinking this but I'm worried that this may result in a different to parse error in which, if `FusedLayerNorm` is somehow used, then some weights will not get initialized properly. I would prefer there being an if-statement catching this and raising an intelligible exception but I'm not sure how to do this without reintroducing the apex dependency. Maybe this?

```suggestion
        if m.__class__.__name__ == 'FusedLayerNorm':
             raise AttributeError("FusedLayerNorm is not supported."):
                 
        elif isinstance(m, nn.LayerNorm):
```